uint8 gACT 
uint8 gGTO 
uint8 gSTA 
uint8 gOBJ 
uint8 gFLT
uint8 gPR
uint8 gPO
uint8 gCU   # gCU, The current is read instantaneously from the motor drive, value between 0x00 and 0xFF, approximate current equivalent is 10 * value read in mA.
